source/common/config/grpc_mux_impl.cc

```diff
@@ -8,14 +8,18 @@
 namespace Envoy {
 namespace Config {
 
-GrpcMuxImpl::GrpcMuxImpl(const envoy::api::v2::Node& node, Grpc::AsyncClientPtr async_client,
+GrpcMuxImpl::GrpcMuxImpl(const envoy::api::v2::core::Node& node, Grpc::AsyncClientPtr async_client,
                          Event::Dispatcher& dispatcher,
                          const Protobuf::MethodDescriptor& service_method)
     : node_(node), async_client_(std::move(async_client)), service_method_(service_method) {
```

Comment:
should we add some kind of an `ASSERT` here on `service_method_`? with out that if the `MethodByName` is not found in generic pool, it just segfaults.

Reply:
Merged to keep things moving, tracking this concern at https://github.com/envoyproxy/envoy/issues/2528.